documentation/Components/LayoutSample.fsx

```diff
@@ -1,11 +1,90 @@
 #load "../../.paket/load/netstandard2.0/main.group.fsx"
 #load "../../src/Common.fs"
-#load "../../src/Layout.fs"
+#load "../../src/Container.fs"
+#load "../../src/Row.fs"
+#load "../../src/Col.fs"
 
 open Fable.Core.JsInterop
 open Fable.React
+open Fable.React.Props
 open Reactstrap
 
-let private layoutSample = FunctionComponent.Of<obj>((fun _ -> fragment [] []), "LayoutSample")
+let container = Container.container
+let row = Row.row
+let col = Col.col
+let custom  attr =
+    attr
+    |> Col.Custom
+
+let private layoutSample =
+    let customLayout =
+        {|
+            size = 6;
+            order = 2;
+            offset = 1;
+        |}
+
+    let customLayout1 =
+        {|
```

Comment:
Use https://github.com/nojaf/fable-reactstrap/blob/master/src/Col.fs#L10 instead or use the `mkCol` helper function.

Reply:
@nojaf not sure `order = !^0` is for?

Reply:
The `!^` will cast any type to a `U2<_,_>`, `U3<_,_,_>`, needed in the signature.
For example `let foo (a: U2<string,int>) = ...`
Can be called by string using `foo (!^ "meh")` or an int using `foo (!^ 42)`